### PR TITLE
add payment api

### PR DESCRIPTION
**Pull request by yorickdevries**

### CODE REVIEW

File: payments/src/main/java/nl/tudelft/wdm/group1/payments/web/PaymentController.java [hunk 1]
```diff
@@ -26,17 +27,34 @@
         this.paymentRepository = paymentRepository;
     }
 
-    @PostMapping
-    public Payment addPayment() {
-        Payment payment = new Payment();
-
-        producer.send(payment);
+    //POST /payments/{user_id}/{order_id}
```
Comment (qurben) [inline payments/src/main/java/nl/tudelft/wdm/group1/payments/web/PaymentController.java]: Redundant comment?

File: payments/src/test/java/nl/tudelft/wdm/group1/payments/PaymentsApplicationTest.java [hunk 2]
```diff
@@ -47,29 +48,44 @@
...
+    private UUID defaultUserId = UUID.randomUUID();
+    private UUID defaultOrderId = UUID.randomUUID();
 
     @Before
-    public void setUp() {
-        defaultPayment = new Payment();
+    public void setUp() throws Exception {
+        defaultPayment = new Payment(defaultUserId, defaultOrderId);
         paymentRepository.add(defaultPayment);
+        assertThat(paymentRepository.find(defaultPayment.getId())).isNotNull();
```
Comment (qurben) [inline payments/src/test/java/nl/tudelft/wdm/group1/payments/PaymentsApplicationTest.java]: Please do not test in the setUp method.

**Comment by yorickdevries** [comment]
Thanks for the feedback, Lets go over it together on tuesday

**Comment by casperboone** [comment]
@yorickdevries Could please include the amount in the payment?

**Pull request merged**